src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -16,6 +17,7 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniqueFlashcardList flashcards;
+    private Attempt currentAttempt;
```

Comment:
should this be moved to quizbook instead?

Reply:
Yep, thanks for pointing that out. 👍  I've changed it.